scripts/ASTImportTest.sh

```diff
@@ -19,9 +19,22 @@
 # (c) solidity contributors.
 # ------------------------------------------------------------------------------
 # Bash script to test the import/exports.
+#
```

Comment:
When do you plan to rename this script, I see it's in the TODO?

Reply:
After @cameel is ok with that script ;) I think that if I rename it now, the diffs are not nicely visible anymore. However, I think I still need to work a bit on that script. I hope I can finish that today.

Reply:
You could also rename it in a separate PR (before or after this one).

Anything you can extract from here and make a separate PR helps :)

Reply:
You could also rename it in a separate PR (before or after this one).

Anything you can extract from here and make a separate PR helps :)